File: Proyectos C++ (3)/Hombres y mujeres.cpp
#include<iostream>

using namespace std;

int main(){

	int num, g, h=0, m=0  ;
	cout<<"Este programa va a determinar cuantos hombres y mujeres hay en un curso de 25 estudiantes"<<endl;
	for(num=1;num<=25;num++){
		cout<<"Ingrese su genero de la siguiente manera Hombre=1, Mujeres=2: "<<endl ;
		cin>>g;
		if(g==1)
			h++;
		else
			m++;
			
	}
	cout<<"Hay " <<h<<" hombres"<<endl;
	cout<<"Hay " <<m<<" mujeres"<<endl;

}	

File: Proyectos Python (3)/Hombres y mujeres.py
print("Determinar numero de hombres y de mujeres en un  grupo de 5 personas")
num = 1
h = 0
m = 0

for num in range (1,26):
    op=int(input("Ingrese su genero 1=h 2=h: "))
    if(op==1):
        h=h+1
        
        
    else:
        m=1+m
        
        
print("Hombres: " ,h, "\nMujeres: " ,m)
